src/test/java/com/google/javascript/clutz/base.js

```diff
@@ -31,3 +31,21 @@ goog.isDef = function(val) {};
 goog.provide('goog.Uri');
 /** @constructor */
 goog.Uri =  function() {}
+
+/**
+ * @param {Object} object Object whose static property is being exported.
+ */
+goog.functionUsingGlobalObject = function(object) {
```

Comment:
given that this is supposed to mirror base, why not just put the actual function that uses Object from goog base.

Reply:
I've replaced my abstractions with the actual versions.